derive_builder_core/src/builder.rs

```diff
@@ -231,15 +237,35 @@ impl<'a> ToTokens for Builder<'a> {
                 let builder_error_ident = format_ident!("{}Error", builder_ident);
                 let builder_error_doc = format!("Error type for {}", builder_ident);
 
+                let validation_error = self
+                    .generate_validation_error
+                    .then_some(quote!(
```

Comment:
This is pretty cool, and I hadn't seen it before. However, if the discussion in #303 does lead to this being on track for a merge, we need to avoid using a method that isn't available to 1.56.0; can you change each use of it to an `if` block?

Reply:
Updated the code then built and ran cargo test using 1.56.0 x86_64.